docs/src/main/sphinx/connector/pushdown-char-varchar-case.fragment

```diff
@@ -0,0 +1,17 @@
+Predicate pushdown support
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector does not push down inequality predicates (such as ``!=``, ``>``,
+or ``<``) on columns with `textual types <../language/types.html#string>`_ like
```

Comment:
this should be a ref link.. not a hardcoded path link reserved for external URLs

Reply:
The target doc doesn't have anchors for me to ref link to, so I've had to add them here.